unic-langid-impl/src/likelysubtags/mod.rs

```diff
@@ -33,7 +33,7 @@ pub fn add_likely_subtags(
                 })
                 .ok();
             if let Some(r) = result {
-                return get_lang_from_parts(tables::LANG_REGION[r].2, None, None, None);
+                return unsafe { get_lang_from_parts(tables::LANG_REGION[r].2, None, None, None) };
             }
```

Comment:
ideally leave a comment on each use of unsafe explaining why it's actually safe to do (in these cases, probably something like "safe because all table entries are well formed")